<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OTS - редактор тестов</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="questions">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Название теста</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QTimeEdit" name="time_limit"/>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Установить ограничение по времени
 (0 - без ограничения)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="test_name"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="prev_page">
        <property name="text">
         <string>&lt;&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_page">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="next_page">
        <property name="text">
         <string>&gt;&gt;&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="editor_menu">
    <property name="title">
     <string>Редактор</string>
    </property>
    <widget class="QMenu" name="create_question">
     <property name="title">
      <string>Новый вопрос</string>
     </property>
     <addaction name="create_single"/>
     <addaction name="create_pick"/>
    </widget>
    <addaction name="create_question"/>
    <addaction name="separator"/>
    <addaction name="new_test"/>
    <addaction name="open_test"/>
    <addaction name="save"/>
    <addaction name="save_as"/>
    <addaction name="separator"/>
    <addaction name="exit_editor"/>
   </widget>
   <addaction name="editor_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="create_single">
   <property name="text">
    <string>С одним ответом</string>
   </property>
  </action>
  <action name="create_pick">
   <property name="text">
    <string>С выбором из нескольких ответов</string>
   </property>
  </action>
  <action name="create_pickmulti">
   <property name="icon">
    <iconset>
     <normalon>:/logos/create_pickmulti.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>С выбором нескольких ответов</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="save_as">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="exit_editor">
   <property name="text">
    <string>Выйти из редактора</string>
   </property>
  </action>
  <action name="testoptions">
   <property name="text">
    <string>Параметры теста</string>
   </property>
  </action>
  <action name="open_test">
   <property name="text">
    <string>Открыть тест</string>
   </property>
  </action>
  <action name="new_test">
   <property name="text">
    <string>Новый тест</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../data/images.qrc"/>
 </resources>
 <connections/>
</ui>
